### ISSUE TITLE
Issue with Min-Max normalization code

### ISSUE THREAD

**Issue by Sithara85** [zhangxiaoyu11/OmiVAE#2]
Hi Dr. Zhang,

I have been using some of your GitHub code to process RNAseq data and to build the VAE classifier model. I noticed that you have used the below code for MIn-Max normalization. 
**# Normalize the dataframe to the range of 0-1
    if norm_0_1:
        # Min-max normalization
        data_df = (data_df - data_df.min().min()) / (data_df.max().max() - data_df.min().min())**

 I think you have to use data_df = (data_df - data_df.min()) / (data_df.max() - data_df.min()) to obtain matching values to sklearn.preprocessing.MinMaxScaler() function. Can you please check?
